Milliner/src/Controller/MillinerController.php

```diff
@@ -44,6 +44,10 @@ public function saveNode($uuid, Request $request)
     {
         $token = $request->headers->get("Authorization", null);
         $jsonld_url = $request->headers->get("Content-Location");
+        $event_type = $request->headers->get("Event");
+        if (empty($event_type)) {
+            $event_type = null;
+        }
```

Comment:
Like `$token` on line 45 above, you can provide a default in the `->get()` which is returned if the Header doesn't exist. 
However my bigger question is: Is "Event" an existing header or is it a brand new one?

If it is brand new, then it should probably be `X-Islandora-Event` to identify it is a non-standard header and fit with our existing [`X-Islandora-Args`](https://github.com/Islandora-CLAW/Crayfish/blob/dev/Houdini/src/Controller/HoudiniController.php#L94)

Reply:
It was a made up header.